### PR TITLE
Translating ShowMore component to portuguese

### PR DESCRIPTION
**Pull request by renatoricardi**
#### What problem is this solving?
There are some "buttons" that show texts in English instead of the region itself.

#### How to test it?
[Workspace](https://aa254--lojausereserva.myvtex.com/camiseta-estampada-r-pica-pau-class0071847/p)

#### Screenshots or example usage:
image
image

### CODE REVIEW

**Review by beatrizmaselli — APPROVED** [review]
LGTM

**Review by beatrizmaselli — CHANGES_REQUESTED** [review]
Add `en.json` static messages and remove defaultMessage

**Comment by Mecrano** [comment]
LGTM

**Review by Mecrano — APPROVED** [review]
LGTM

File: messages/en.json [hunk 1]
```diff
@@ -125,5 +125,7 @@
   "store/reviews.list.filterOptions.two-stars": "2 stars",
   "store/reviews.list.filterOptions.three-stars": "3 stars",
   "store/reviews.list.filterOptions.four-stars": "4 stars",
-  "store/reviews.list.filterOptions.five-stars": "5 stars"
+  "store/reviews.list.filterOptions.five-stars": "5 stars",
+  "store/reviews.list.showMore": "Show More",
+  "store/reviews.list.showLess": "Show Less"
```
Comment (camilavcoutinho) [inline messages/en.json:130]:
```suggestion
  "store/reviews.list.showMore": "Show more",
  "store/reviews.list.showLess": "Show less"
```

**Review by camilavcoutinho — CHANGES_REQUESTED** [review]

**Review by camilavcoutinho — APPROVED** [review]

**Review by beatrizmaselli — APPROVED** [review]
LGTM

**Review by beatrizmaselli — APPROVED** [review]

**Pull request merged**